### PR TITLE
Feature/internet extension

### PR DESCRIPTION
**Pull request by GautierDele**
Doesn't contain user agent

### CODE REVIEW

File: src/Extensions/InternetExtension.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    public function domain(): string
+    {
+        return sprintf('%s.%s', $this->sdl(), $this->tld());
+    }
+
+    public function ipv4(): string
+    {
+        return long2ip($this->randomizer->getInt(0, 1) === 0 ? $this->randomizer->getInt(-2147483648, -2) : $this->randomizer->getInt(16777216, 2147483647));
```
Comment (martinsoenen) [inline src/Extensions/InternetExtension.php:29]: The negative numbers ensures compatibility with 32 bits systems, but I think we should check all the positive numbers to ensure compatibility with 64 bits systems. Cause `ip2long` (invert of `long2ip`) always return positive numbers so a lot of 64 bits systems are storing IPs as int from `0` to `4294967295`

File: src/Extensions/InternetExtension.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    public function ipv6(): string
+    {
+        $res = [];
+
+        for ($i = 0; $i < 8; $i++) {
+            $res[] = dechex($this->randomizer->getInt(0, 65535));
```
Comment (martinsoenen) [inline src/Extensions/InternetExtension.php:37]: Do you prefer this way to ensure compatibility, or to use `inet_ntop` with a randomizer, which is the prefered way of using IPv6 when PHP is compiled with IPv6 ?

**Review by martinsoenen — COMMENTED** [review]
Here are some questions about IPs

File: src/Extensions/InternetExtension.php (hunk 1 above)
Comment (GautierDele) [reply src/Extensions/InternetExtension.php:29]: It seems you are right, the documentation is poor on that side, I don't know if I should be avoiding local ips, like in this case, what do you think ?

File: src/Extensions/InternetExtension.php (hunk 2 above)
Comment (GautierDele) [reply src/Extensions/InternetExtension.php:37]: I did have a look at this but i'm not familiar enough with this function and documentation didn't help me on that, but this isn't a reason, will need to have a look

**Comment by GautierDele** [comment]
@martinsoenen should be fixed 😉

**Pull request merged**